data/registry/tools-elixir-tails.yml

```diff
@@ -0,0 +1,17 @@
+title: tails
+registryType: utilities
+language: elixir
+tags:
+  - otel
+  - webserver
+  - tailing
+urls:
+  repo: https://github.com/jaronoff97/tails
+license: Apache 2.0
+description:
+  Tails is a small webserver written in Elixir that listens on a socket and
+  streams live messages from a collector.
+authors:
+  - name: jaronoff97
```

Comment:
github this is my handle, you know who i am :sob:

Reply:
```suggestion
  - name: Jacob Aronoff
```

Reply:
fixed